src/app/services/authorization.service.ts

```diff
@@ -46,7 +47,7 @@ export class AuthorizationService {
           console.log('Token: '+ e.newValue);
           this.authToken.value = e.newValue;
           window.removeEventListener('storage', storageChanged, false);
-
+          this.spotifyService.token = e.newValue;
```

Comment:
This shall be handled in #19 and #14, we do need to figure the best way to save the token.